roles/manifest/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Whether to download manifest. When True, it requires access.redhat.com login and Manifest UUID to be specified.
+download_manifest: False
+
+# Red Hat Customer Portal username and password (i.e. access.redhat.com) login and Manifest UUID.
+rhsm_user:
+rhsm_password:
+manifest_uuid:
+
+# Foreman username and password
+foreman_username: admin
+foreman_password: changeme
+
+# Foreman server URL
+foreman_server_url: https://foreman.example.com
+
+# Foreman organization
+foreman_organization: "Default Organization"
```

Comment:
I'd like to consider introducing a role for common variables that is a dependency of other roles so as to define the variables in a common place. Naming is hard for such a role:

 * server_variables
 * api_variales
 * common_variables
 * ?!?!!